src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml

```diff
@@ -28,11 +30,179 @@
             <Setter Property="BorderThickness" Value="1"/>
             <Setter Property="FontSize" Value="12"/>
         </Style>
-        <converters:BooleanToVisibilityConverter x:Name="BooleanToVisibilityConverter"/>
         <converters:BooleanToVisibilityNegationConverter x:Name="BooleanToVisibilityNegationConverter"/>
+
+        <Style x:Key="SwitchModeToggleButtonStyle" TargetType="ToggleButton">
```

Comment:
In high contrast mode the toggled state has a blue background with a blue foreground in High Contrast # 2

Reply:
This is expected (it's the default style of ToggleButton)